src/OpenTelemetry.Instrumentation.AWSLambda/OpenTelemetry.Instrumentation.AWSLambda.csproj

```diff
@@ -1,19 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
Given .NET Core 3.1 is well out of support is there a reason .NET Standard 2.0 still needs to be targeted? That would allow you to get rid of some if the conditional compilation blocks and reduce the complexity.

Reply:
Actually a good point. We did want to get rid of netstandard2.0 already, but delayed it because we wanted to have a particular bugfix in a backwards compatible release. But by now I think we could upgrade to at least `netcoreapp3.1` as done in #1270, although I also see no reason not to upgrade to 6.0 (AWS already has that as minimum).

Would be great if we could get rid of all the `#ifdef`s and have only a single target framework specifier.

Reply:
I'd prefer to not do that in the same PR.

Would you rather:
1. I do another PR to change `netstandard2.0` to `net6.0` and then rebase this one on top of it.
2. Merge this (once feedback addressed) as-is, then I do a follow-up to remove `netstandard2.0`.

Reply:
#1545